src/searcher/censys.ts

```diff
@@ -18,22 +18,29 @@ export class Censys implements Searcher {
   public searchByASN(query: string): string {
     const asn = extractASNumber(query);
     return buildURL(this.baseURL, "/search", {
-      q: `autonomous_system.asn:${asn}`,
+      q: `autonomous_system.asn: ${asn}`,
```

Comment:
Does it really need a space after the search filter?

Reply:
Not necessarily, its more stylistic.